### PR TITLE
Complete https://github.com/p4lang/p4runtime/issues/13 to fix interna…

### PR DESCRIPTION
**Pull request by chrispsommers**
…l hyperlinks.

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -996,9 +996,8 @@ should support for the table.  See the "Size" subsection within the "Table
 Properties" section of the P4_16 language specification version 1.1.0 for
 details.
 
-* idle_timeout_behavior, which describes the behaviour of the data plane when
-the idle timeout of a table entry expires (see [idle- imeout
-section](#heading=h.ltifp7ix124)). Value can be any of the IdleTimeoutBehavior
+* idle_timeout_behavior, which describes the behavior of the data plane when
+the idle timeout of a table entry expires (see [Idle-Timeout](#sec-idle-timeout) section). Value can be any of the IdleTimeoutBehavior
```
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk]: that looks like more than 80 characters

File: docs/v1/P4Runtime-Spec.mdk [hunk 2]
```diff
@@ -2321,8 +2321,7 @@ defined as Direct or Indirect (indexed) instances.
 
 ### DirectCounterEntry
 
-A direct counter is a direct resource associated with a TableEntry (see [9.1.5
-Direct resources](#heading=h.3rjuxc1zdvon)). The counter_data field of the
+A direct counter is a direct resource associated with a TableEntry (see [Direct resources](#sec-direct-resources)). The counter_data field of the
```
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk]: same, more than 80 characters

File: docs/v1/P4Runtime-Spec.mdk [hunk 3]
```diff
@@ -2440,8 +2439,7 @@ The protobuf fields are defined as follows:
 
 ### DirectMeterEntry
 
-A direct meter is a direct resource associated with a TableEntry (see [9.1.5
-Direct resources](#heading=h.3rjuxc1zdvon)). The meter_config field of the
+A direct meter is a direct resource associated with a TableEntry (see [Direct resources](#sec-direct-resources)). The meter_config field of the
```
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk]: same

Comment (chrispsommers) [reply docs/v1/P4Runtime-Spec.mdk]: Good feedback, thanks.

**Comment by antoninbas** [comment]
Fixes #13

**Pull request merged**